src/odemis/acq/stream/_base.py

```diff
@@ -561,6 +611,77 @@ def _getDetectorVA(self, vaname):
                 raise AttributeError(u"Detector has not VA %s" % (vaname,))
             return hwva
 
+    def _linkHwAxes(self):
+        """"
+        Link the axes to the hardware components
+        If local axis Vigilant Attributes's are specified, write the values of the local axis VA
+        to the real hardware
+        """
+
+        if hasattr(self, "axis_map") and self.axis_map:
+
+            logging.info("Moving linked HW axes")
+
+            moving_axes = []
+            for ax_name in self.axis_map:
```

Comment:
You should be able to simplify by writing: `for va_name, (axis_name, actuator) in self.axis_map.item():`